src/ga4gh/vrs/extras/annotator/vcf.py

```diff
@@ -59,6 +52,19 @@ def default_value(self) -> Literal[".", -1]:
     }
 )
 
+# Short State.sequence will be included in output VCF if <= this value,
+# otherwise output will be emitted as the "." character.
+MAX_LITERAL_STATE_LENGTH = 15
+
+
+def _sequence_value_for_info(sequence: str | sequenceString) -> str | None:
+    """Return a literal sequence suitable for INFO fields if it is short enough."""
+    if not sequence:
+        return None
+    seq = getattr(sequence, "root", sequence)
+    seq_str = str(seq)
+    return seq_str if len(seq_str) <= MAX_LITERAL_STATE_LENGTH else None
```

Comment:
Don't think we need to cast as a string (redundant)

```suggestion
    return seq if len(seq) <= MAX_LITERAL_STATE_LENGTH else None
```